src/languageSetup.ts

```diff
@@ -67,7 +67,13 @@ export async function activateLanguageServer(context: vscode.ExtensionContext, s
     status.dispose();
     
     const startScriptPath = customPath || path.resolve(langServerInstallDir, "server", "bin", correctScriptName("kotlin-language-server"));
-    const options = { outputChannel, startScriptPath, tcpPort, env };
+
+    const storagePath = context.storagePath
+    if (!fs.existsSync(storagePath)){
+        fs.mkdirSync(storagePath);
+    }
```

Comment:
I think we could use the `async` equivalents, which I'd prefer if possible. Off the top of my head (could be wrong):

```suggestion
    if (!(await fs.promises.exists(storagePath))) {
        await fs.promises.mkdir(storagePath);
    }
```